### PR TITLE
user cluster token and namespace 

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: fandogh_cli/cluster_commands.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 from .config import *
 
-fandogh = [{'name': 'fandogh', 'url': 'https://api.fandogh.cloud', 'active': True}]
+fandogh = [{'name': 'fandogh', 'url': 'fandogh.cloud', 'active': True}]
```
Comment (Anonycoders) [inline fandogh_cli/cluster_commands.py:6]: what happens to old version which have fandogh https://api.fandogh.cloud in credentials file?

Comment (Anonycoders) [inline fandogh_cli/cluster_commands.py:6]: what happens if user try to add cluster with url starting with https:// or https://api.... ?

File: fandogh_cli/namespace_commands.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import click
-from fandogh_cli.config import get_user_config
+from fandogh_cli.config import get_user_config, set_cluster_namespace
 
 from fandogh_cli.fandogh_client.namespace_client import *
 from .base_commands import FandoghCommand
```
Comment (Anonycoders) [inline fandogh_cli/namespace_commands.py:5]: namespace list command does not show activated namespace after changing cluster.

File: fandogh_cli/cluster_commands.py [hunk 3]
```diff
@@ -16,6 +16,9 @@ def cluster():
 @click.option('--name', prompt='name', help="Enter name for cluster url")
 @click.option('--url', prompt='url', help="Enter cluster URL")
 def add(name, url):
+    if url.startswith("http"):
+        click.echo("Please enter URL without https or http just use domain url like 'FANDOGH.CLOUD'")
```
Comment (s-soroosh) [inline fandogh_cli/cluster_commands.py]: like 'fandogh.cloud'

File: fandogh_cli/config.py [hunk 4]
```diff
@@ -63,17 +63,54 @@ def get_user_config():
...
+        if cluster['active']:
+            del clusters[index]
+            custom_dict = [
+                dict(name=cluster['name'], url=cluster['url'], active=True, token=token),
+                *clusters]
+            set_cluster_config(custom_dict)
+
+
+def get_cluster_namespace():
+    if clusters:
```
Comment (s-soroosh) [inline fandogh_cli/config.py]: what if not?

File: fandogh_cli/config.py [hunk 5]
```diff
@@ -63,17 +63,54 @@ def get_user_config():
...
+        for index, cluster in enumerate(clusters):
```
Comment (s-soroosh) [inline fandogh_cli/config.py]: write a function `get_active_cluster()` and use it everywhere.

File: fandogh_cli/config.py [hunk 6]
```diff
@@ -63,17 +63,54 @@ def get_user_config():
...
+def set_cluster_namespace(namespace):
+    if clusters:
+        for index, cluster in enumerate(clusters):
+            if cluster['active']:
+                token = cluster.get('token', None)
+                del clusters[index]
+                custom_dict = [
+                    dict(name=cluster['name'], url=cluster['url'], active=True, token=token,
+                         namespace=namespace),
+                    *clusters]
```
Comment (s-soroosh) [inline fandogh_cli/config.py]: *clusters? does it work in python2

**Pull request merged**